pkg/distributor/distributor_ring.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/cortexproject/cortex/pkg/util/flagext"
 )
 
+var (
+	RingNameForClient = "distributor"
+)
+
```

Comment:
```suggestion
const (
	ringNameForClient = "distributor"
)
```

And move to `distributor_ring.go` where it is used.

(Alternatively, just inline it)